_data/publications/1752736.yml

```
@@ -2,4 +2,7 @@ inspire-id: 1752736
 project: 
 - madminer
 - exploratory-ml
+link: https://doi.org/10.3847/1538-4357/ab4c41
+date: November 19, 2019
```

Dates in ISO 8061 format, please
```suggestion
date: 2019-11-19
```

---

Ok, should probably update instructions. Example is `date: Dec 10, 2018`
https://iris-hep.org/docs/add_publication.html

Also, would be nice if there was an example for how the publication is rendered. Not always clear

---

I had a sneaking suspicion that the example was probably wrong...

---

What do you mean by an example of rendering? I'd be happy to add one.

---

I updated the file in the PR. I didn't see the "Commit suggestion" option until now. Both files updated.